### PR TITLE
Add example images of simulation of color deficiency

### PR DESCRIPTION
**Pull request by kimikage**
cf. https://github.com/JuliaGraphics/Colors.jl/issues/398#issuecomment-596120539

### CODE REVIEW

**Comment by kimikage** [comment]
The sample image included in this PR is the tricky  version with reduced colors and sizes. The original version is as follows (CC0).
beads_1920x1080

**Comment by johnnychen94** [comment]
I like the colors 😍

**Comment by kimikage** [comment]
> may I add this to TestImages?

Sure. It is released under the Creative Commons CC0 license.

**Comment by kimikage** [comment]
I forgot to erase unnecessary code, so I removed it.:sweat_smile:

Since this changes the docs only, I will merge this within a few days if there is no objection.

**Comment by timholy** [comment]
It seems likely to be useful for many purposes, not just colors but segmentation algorithms, algorithms inferring depth from blur, etc. And it's just plain beautiful. Thanks, @kimikage!

Out of curiosity, why is it an svg? Did you actually generate this image rather that it being a photograph?

**Comment by kimikage** [comment]
> Out of curiosity, why is it an svg? Did you actually generate this image rather that it being a photograph?

This is a 3d computer-generated image, and post-processed to make some glass beads to be in indistinguishable color pairs.

I hated depending on ImageMagick just for these examples, so I decided to use the technique used in the cross-sectional charts (cf. PR #406), i.e. the use of paletted PNGs. However, 256 colors are too few to represent this image.
And here's the thing. :smile:

beads_y
beads_g
beads_r

We shouldn't really use the dithering because of the different color spaces from normal vision. Therefore, I adjusted the hues so that they did not deviate significantly.

**Comment by kimikage** [comment]
I made a mistake on the HTML syntax. :sob:
I don't know whether the cause is it or the embedded PNG, but the github pages hasn't been updated.

File: docs/sampleimages.jl [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+function BeadsImageSVG(caption::String; filter=nothing, width="64mm", height="36mm")
+    io = IOBuffer()
+    id = string(hash(caption), base=16)
+
+    open(joinpath("assets", "figures", "beads.svg"), "r") do file
+        for line in eachline(file, keep=true)
+            occursin("<?xml", line) && continue
+            if occursin("<svg", line)
+                line = replace(line, ">"=>
+                    """width="$width" height="$height" style="display:inline; margin-left:1em; margin-bottom:1em">""")
```
Comment (kimikage) [inline docs/sampleimages.jl]:
```diff
-"""width="$width"
+""" width="$width"
```

**Comment by kimikage** [comment]
Updated! :smile:
http://juliagraphics.github.io/Colors.jl/latest/advancedfunctions/
(triggerd by PR #373)

**Comment by timholy** [comment]
The new docs just look fantastic.

**Comment by johnnychen94** [comment]
FYI, Documenter 0.24 provides a `push_preview` keyword for non-fork PRs

For example, https://github.com/JuliaImages/juliaimages.github.io/pull/115

**Pull request merged**